### ISSUE TITLE
https://sharespark.net/index.php, amazon ads not blocked

### ISSUE THREAD

**Issue by rajatmathurs** [uBlockOrigin/uBlock-issues#159]
### Prerequisites

- [ ] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [ ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [x] uBlock Origin is the only extension
    - [x] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin
- [x] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

When you open the website, amazon ads not blocked on home page and every other pages of the site

[Description of the bug or feature]

https://sharespark.net/index.php

[A specific URL is MANDATORY for issue happening on a web page, even if it happens "everywhere"]

### Steps to Reproduce

1. Open https://sharespark.net/index.php, you will see amazon ads
2. click on any link
3. Every page is showing amazon ads

### Expected behavior:

Ads should be blocked

### Actual behavior:

[What actually happened]

### Supporting Evidence

https://i.imgur.com/N2vEAxM.png

### Your environment

* uBlock Origin version: v1.16.14
* Browser Name and version: Firefox 52.9.0
* Operating System and version: Windows XP SP3

**Comment by uBlock-user** [comment]
Duplicate of https://github.com/uBlockOrigin/uBlock-issues/issues/158
